apps/ui/src/views/Space.vue

```diff
@@ -17,25 +18,21 @@ const space = computed(() => {
 });
 
 watch(
-  [resolved, networkId, address],
-  async ([resolved, networkId, address]) => {
-    if (!resolved || !networkId || !address) return;
+  [resolved, networkId, address, () => web3.value.account],
+  async ([resolved, networkId, address, account]) => {
+    if (!resolved || !networkId || !address || !account) return;
```

Comment:
Space doesn't load if no account is not logged in
<img width="1175" alt="Untitled 4" src="https://github.com/user-attachments/assets/1889a901-a752-412d-8055-66e4ee1b8f19">

Reply:
Nice catch